CIPs/cip-xxxx.md

```diff
@@ -0,0 +1,128 @@
+---
+cip: <to be assigned>
+title: Processes Around Granda Mento Exchanges
+author: Trevor Porter (@tkporter)
+discussions-to: <URL>
+status: Draft
+type: Meta
```

Comment:
It's not a Meta CIP, a Meta CIP contains multiple CIPs in it. This is more of an Informational CIP given its around processes.

Reply:
From https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0000.md:
```
* A Meta CIP describes a process surrounding Celo or proposes a change to (or an event in) a process. Meta CIPs are like Standards Track CIPs but apply to areas other than the Celo protocol itself. They may propose an implementation, but not to Celo’s codebase; they often require community consensus; unlike Informational CIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include code of conduct updates, procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Celo development. Any Meta CIP is also considered a Process CIP.
* An Informational CIP describes a Celo design issue, or provides general guidelines or information to the Celo community, but does not propose a new feature. Informational CIPs do not necessarily represent Celo community consensus or a recommendation, so users and implementers are free to ignore Informational CIPs or follow their advice.
```

I don't see anything about a Meta CIP containing multiple CIPs?
This CIP feels to me like it falls somewhere in the category of `procedures, guidelines, changes to the decision-making process`. Users are not really free to ignore these processes and I believe this CIP requires Celo community consensus.

Reply:
Good point, sounds good